src/MoBi.Presentation/Presenter/IndividualBuildingBlockPresenter.cs

```diff
@@ -45,5 +49,19 @@ public bool HasAtLeastOneValue(int pathElementIndex)
       {
          return _individualBuildingBlockDTO.Parameters.HasAtLeastOneValue(pathElementIndex);
       }
+
+      public void Handle(EntitySelectedEvent eventToHandle)
+      {
+         if (eventToHandle.ObjectBase is IndividualParameter individualParameter)
+         {
+            if (_buildingBlock.Contains(individualParameter))
```

Comment:
_buildingBlock here is acceptable (even tho I would prefer individual) because we are in an IndBBPresenter